twisted/python/compat.py

```diff
@@ -756,6 +759,25 @@ def _coercedUnicode(s):
 
 
 
+def _maybeMBCS(s):
+    """
+    Convert the string C{s} to a L{unicode} string, if required.
+
+    @param s: The string to convert.
+    @type s: L{bytes} or L{unicode}
+
+    @return: The string, decoded using MBCS if needed.
```

Comment:
I feel like this *really* needs a [`@raise`](https://epydoc.sourceforge.net/epytext.html) explaining that MBCS is definitely *not* a sure-fire encoding solution.